From e9d5a380ee601eb84fe0a0caf587fd8fbca9c220 Mon Sep 17 00:00:00 2001
From: epalrov <paolo.rovelli@ericsson.com>
Date: Tue, 12 Apr 2016 09:34:03 +0200
Subject: [PATCH 1/2] fix siae

---
 netconf/src/agt/agt_val.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/netconf/src/agt/agt_val.c b/netconf/src/agt/agt_val.c
index ba82369..9673dcd 100644
--- a/netconf/src/agt/agt_val.c
+++ b/netconf/src/agt/agt_val.c
@@ -3688,6 +3688,9 @@ static status_t
     errinfo = NULL;
     typdef = obj_get_typdef(val->obj);
 
+    if (scb == NULL) //SIAE
+        return res; //SIAE
+
     switch (val->btyp) {
     case NCX_BT_LEAFREF:
         /* do a complete parsing to retrieve the
-- 
1.9.1

